### PR TITLE
Browse

### PR DESCRIPTION
**Pull request by CanOfBees**
## What does this do? ##
This PR adds updated MODS, where matching `subject/topic` values corresponding to the Browse spreadsheet have new `note[@displayLabel='Browse']` elements added.

### CODE REVIEW

**Review by markpbaggett — APPROVED** [review]
Looks good to me.

**Review by markpbaggett — CHANGES_REQUESTED** [review]
Wait, we missed locations. Let's get that first.

File: cleaned_data/remediation_files/addtowns.xsl [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <note displayLabel="Browse"><xsl:value-of select="'Arrowmont School of Arts &amp; Crafts'"/></note>
+  </xsl:template>
+  
+  <!-- GSMNP -->
+  <xsl:template match="subject[@valueURI='http://id.loc.gov/authorities/subjects/sh2005004887']">
+    <xsl:copy>
+      <xsl:apply-templates select="@*|node()"/>
+    </xsl:copy>
+    <note displayLabel="Browse"><xsl:value-of select="'Great Smoky Mountains National Park'"/></note>
+  </xsl:template>
```
Comment (markpbaggett) [inline cleaned_data/remediation_files/addtowns.xsl]: We don't need these templates or their output.  This is already limitable in the application.

File: cleaned_data/modsbyPID/rfta:98.xml [hunk 2]
```diff
@@ -46,13 +46,15 @@
...
    </subject>
+   <note displayLabel="Browse">Emergency Management</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:98.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:91.xml [hunk 3]
```diff
@@ -52,6 +52,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:91.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:87.xml [hunk 4]
```diff
@@ -50,13 +50,15 @@
...
    </subject>
+   <note displayLabel="Browse">Disaster Response &amp; Recovery</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:87.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:85.xml [hunk 5]
```diff
@@ -56,6 +57,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:85.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:82.xml [hunk 6]
```diff
@@ -48,13 +49,15 @@
...
    </subject>
+   <note displayLabel="Browse">Ecology</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:82.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:74.xml [hunk 7]
```diff
@@ -60,6 +60,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:74.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:71.xml [hunk 8]
```diff
@@ -56,6 +56,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:71.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:70.xml [hunk 9]
```diff
@@ -46,13 +46,15 @@
...
    </subject>
+   <note displayLabel="Browse">Disaster Response &amp; Recovery</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:70.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:65.xml [hunk 10]
```diff
@@ -45,13 +45,15 @@
...
    </subject>
+   <note displayLabel="Browse">Ecology</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:65.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:52.xml [hunk 11]
```diff
@@ -51,6 +51,7 @@
          <coordinates>35.712925, -83.5104784</coordinates>
       </cartographics>
    </subject>
+   <note displayLabel="Browse">Arrowmont School of Arts &amp; Crafts</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:52.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:50.xml [hunk 12]
```diff
@@ -51,6 +51,7 @@
          <coordinates>35.712925, -83.5104784</coordinates>
       </cartographics>
    </subject>
+   <note displayLabel="Browse">Arrowmont School of Arts &amp; Crafts</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:50.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:49.xml [hunk 13]
```diff
@@ -57,6 +58,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:49.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:38.xml [hunk 14]
```diff
@@ -49,6 +49,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:38.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:122.xml [hunk 15]
```diff
@@ -54,6 +54,7 @@
          <coordinates>35.712925, -83.5104784</coordinates>
       </cartographics>
    </subject>
+   <note displayLabel="Browse">Arrowmont School of Arts &amp; Crafts</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:122.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:13.xml [hunk 16]
```diff
@@ -72,6 +73,7 @@
          <coordinates>35.712925, -83.5104784</coordinates>
       </cartographics>
    </subject>
+   <note displayLabel="Browse">Arrowmont School of Arts &amp; Crafts</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:13.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:155.xml [hunk 17]
```diff
@@ -68,13 +68,15 @@
...
    </subject>
+   <note displayLabel="Browse">Disaster Response &amp; Recovery</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:155.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:22.xml [hunk 18]
```diff
@@ -49,6 +50,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:22.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:23.xml [hunk 19]
```diff
@@ -54,6 +55,7 @@
          <coordinates>35.712925, -83.5104784</coordinates>
       </cartographics>
    </subject>
+   <note displayLabel="Browse">Arrowmont School of Arts &amp; Crafts</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:23.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:25.xml [hunk 20]
```diff
@@ -42,13 +42,15 @@
...
    </subject>
+   <note displayLabel="Browse">Ecology</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:25.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:26.xml [hunk 21]
```diff
@@ -42,13 +42,15 @@
...
    </subject>
+   <note displayLabel="Browse">Ecology</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:26.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:27.xml [hunk 22]
```diff
@@ -41,13 +41,15 @@
...
    </subject>
+   <note displayLabel="Browse">Ecology</note>
    <subject authority="lcsh"
             valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
       <topic>Wildfires</topic>
    </subject>
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:27.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:28.xml [hunk 23]
```diff
@@ -48,6 +48,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:28.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:11.xml [hunk 24]
```diff
@@ -48,6 +48,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:11.xml]:
```suggestion
```

File: cleaned_data/modsbyPID/rfta:10.xml [hunk 25]
```diff
@@ -56,6 +57,7 @@
    <subject valueURI="http://id.loc.gov/authorities/subjects/sh2005004887">
       <geographic>Great Smoky Mountains National Park (N.C. and Tenn.)</geographic>
    </subject>
+   <note displayLabel="Browse">Great Smoky Mountains National Park</note>
```
Comment (markpbaggett) [inline cleaned_data/modsbyPID/rfta:10.xml]:
```suggestion
```

**Review by markpbaggett — CHANGES_REQUESTED** [review]

File: cleaned_data/remediation_files/addtowns.xsl (hunk 1 above)
Comment (markpbaggett) [inline cleaned_data/remediation_files/addtowns.xsl]:
```suggestion
```

**Review by markpbaggett — CHANGES_REQUESTED** [review]

File: cleaned_data/remediation_files/addtowns.xsl (hunk 1 above)
Comment (CanOfBees) [reply cleaned_data/remediation_files/addtowns.xsl]: Thanks for catching this. Overeager templating.

**Review by markpbaggett — APPROVED** [review]
:shipit:

**Comment by CanOfBees** [comment]
@mlhale7 Apologies, I was meant to add you as a reviewer on this draft PR, but I forgot. Sorry for causing duplicative work. Closing this draft.

**Comment by mlhale7** [comment]
@CanOfBees These specific fields will make migration harder down the road, but they will work. We likely should've dropped our LC headings altogether in favor of local fields if that's what is necessary for display.

**Pull request closed without merging**